v2/pkg/engine/datasource/grpc_datasource/execution_plan.go

```diff
@@ -2,16 +2,24 @@ package grpcdatasource
 
 import (
 	"fmt"
+	"slices"
 	"strings"
 
 	"github.com/wundergraph/graphql-go-tools/v2/pkg/ast"
+	"github.com/wundergraph/graphql-go-tools/v2/pkg/astvisitor"
 	"github.com/wundergraph/graphql-go-tools/v2/pkg/engine/plan"
 )
 
 const (
 	// knownTypeOptionalFieldValueName is the name of the field that is used to wrap optional scalar values
 	// in a message as protobuf scalar types are not nullable.
 	knownTypeOptionalFieldValueName = "value"
+
+	// resolverContextDirectiveName is the name of the directive that is used to configure the resolver context.
```

Comment:
I could not find documentation on what does "connect__fieldResolver" do. Is it documented somewhere else?